### PR TITLE
[BUG] removed outparam getters, fixed sensor bug

### PR DESCRIPTION
**Pull request by NPIPHI**
Outparam getters are less clear and readable than return by value getters. Also fixed bug in sensors that assumed a function used an out parameter when it didn't.

### CODE REVIEW

File: includes/Rocket.h [hunk 1]
```diff
@@ -85,32 +56,36 @@ class Rocket {
 
     Vector3d get_Cp_vect() const { return Cp_vect_; };
 
+    std::array<double, 9> get_I() const { return I_; };
+
     /************* Set parameters (all passed by reference) ***************/
```
Comment (AyberkY) [inline includes/Rocket.h:61]: "all passed by reference" no longer accurate

File: src/Sensor.cpp [hunk 2]
```diff
@@ -86,8 +85,7 @@ Accelerometer::Accelerometer(std::string name, Rocket& rocket,
 
 void Accelerometer::update_data(double tStep) {
     if ((tStep - last_update_tStep_) >= (1 / refresh_rate_)) {
-        rocket_.get_r_ddot(data_);
-        rocket_.i2r(data_);
+        rocket_.i2r(rocket_.get_r_ddot());
```
Comment (AyberkY) [inline src/Sensor.cpp:88]: Forgot to assign this to `data_`

**Review by AyberkY — CHANGES_REQUESTED** [review]
The change in `Accelerometer::get_data()` forgets to assign the acceleration vector to the class member `data_`.

I fixed that and ran the sim again, but I still see weird behavior in the sensor plot:
image

^ Not sure what's going on here

**Comment by NPIPHI** [comment]
The plots look like that because Accelerometer logs data in rocket reference frame while the true values are logged in inertial reference frame.

**Comment by NPIPHI** [comment]
Accelerometer also shouldn't feel acceleration from gravity when its flying but that's a different issue

**Pull request merged**